daft/utils.py

```diff
@@ -87,6 +87,13 @@ def map_operator_arrow_semantics_bool(
     ]
 
 
+def python_list_membership_check(
+    left_pylist: list,
+    right_pylist: list,
+) -> list:
+    return [elem in right_pylist for elem in left_pylist]
```

Comment:
Favor using a set

Reply:
Will first try set and fallback to list if the objects are not hashable, implemented in latest commit